docs/apis/mana.md

```diff
@@ -0,0 +1,785 @@
+
+### Parameters
+None.
+
+### Examples
+
+#### cURL
+
+```shell
+curl http://localhost:8080/mana/access/online \
+-X GET \
+-H 'Content-Type: application/json'
+```
+
+#### client lib - `GetOnlineAccessMana()`
+
+```go
+// online access mana
+accessMana, err := goshimAPI.GetOnlineAccessMana()
+if err != nil {
+    // return error
+}
+
+for _, m := accessMana.Online {
+    fmt.Println("full node ID: ", m.ID, "mana rank: ", m.OnlineRank, "access mana: ", m.Mana)
+}
+```
+
+### Response examples
+```shell
+{
+  "online": [
+      {
+        "rank": 1,
+        "shortNodeID": "4AeXyZ26e4G",
+        "nodeID": "2GtxMQD94KvDH1SJPJV7icxofkyV1njuUZKtsqKmtux5",
+        "mana": 75
+      }
+  ],  
+  "timestamp": 1614924295
+}
+```
+
+### Results
+|Return field | Type | Description|
+|:-----|:------|:------|
+| `online`  | OnlineNodeStr | The access mana information of online nodes.   |
+| `timestamp` | int64 | The timestamp of mana updates.  |
+
+#### `OnlineNodeStr`
+|Field | Type | Description|
+|:-----|:------|:------|
+| `rank`  | int | The rank of a node. |
+| `shortNodeID`  | string | The short ID of a node.   |
+| `nodeID`   | string | The full ID of a node.     |
+| `mana`   | float64 | The amount of access mana.     |
+
+
+## `/mana/consensus/online`
+
+You can get a sorted list of online consensus mana of nodes, sorted from the highest consensus mana to the lowest. The highest consensus mana node has OnlineRank 1, and increases 1 by 1 for the following nodes.
+
+### Parameters
+None.
+
+### Examples
+
+#### cURL
+
+```shell
+curl http://localhost:8080/mana/consensus/online \
+-X GET \
+-H 'Content-Type: application/json'
+```
+
+#### client lib - `GetOnlineConsensusMana()`
+
+```go
+// online access mana
+accessMana, err := goshimAPI.GetOnlineConsensusMana()
+if err != nil {
+    // return error
+}
+
+for _, m := accessMana.Online {
+    fmt.Println("full node ID: ", m.ID, "mana rank: ", m.OnlineRank, "consensus mana: ", m.Mana)
+}
+```
+
+### Response examples
+```shell
+{
+  "online": [
+      {
+        "rank": 1,
+        "shortNodeID": "4AeXyZ26e4G",
+        "nodeID": "2GtxMQD94KvDH1SJPJV7icxofkyV1njuUZKtsqKmtux5",
+        "mana": 75
+      }
+  ],  
+  "timestamp": 1614924295
+}
+```
+
+### Results
+|Return field | Type | Description|
+|:-----|:------|:------|
+| `online`  | OnlineNodeStr | The consensus mana information of online nodes.   |
+| `timestamp` | int64 | The timestamp of mana updates.  |
+
+#### `OnlineNodeStr`
+|Field | Type | Description|
+|:-----|:------|:------|
+| `rank`  | int | The rank of a node. |
+| `shortNodeID`  | string | The short ID of a node.   |
+| `nodeID`   | string | The full ID of a node.     |
+| `mana`   | float64 | The amount of consensus mana.     |
+
+
+## `/mana/access/nhighest`
+
+You can get the N highest access mana holders in the network, sorted in descending order.
```

Comment:
maybe we can mention here that if N=0, all nodes that have access mana are returned sorted.